flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FileStoreSource.java

```diff
@@ -130,4 +119,42 @@ public SplitEnumerator<FileStoreSourceSplit, PendingSplitsCheckpoint> restoreEnu
             return new StaticFileStoreSplitEnumerator(context, snapshot, splits);
         }
     }
+
+    private DataFilePlan batchScanSplits(DataTableScan scan) {
+        return scan.plan();
+    }
+
+    private DataFilePlan continuousFirstScanSplits(
+            DataTableScan scan, SnapshotManager snapshotManager) {
+        switch (startupMode) {
+            case FULL:
+                DataFilePlan plan;
+                if (table.options().changelogProducer()
+                        == CoreOptions.ChangelogProducer.FULL_COMPACTION) {
+                    // Read the results of the last full compaction.
+                    // Only full compaction results will appear on the max level.
+                    plan = scan.withLevel(table.options().numLevels() - 1).plan();
+                } else {
+                    plan = scan.plan();
+                }
+                return plan;
+            case LATEST:
```

Comment:
Could the `LATEST` change to `INCREMENTAL` or the `FULL` change to `EARLIEST`? IMO, the `LATEST` and `EARLIEST` are a pair, meanwhile the `INCREMENTAL` and `FULL` are a pair. At present, the `FULL` and `LATEST` are abrupt for users.

Reply:
This is from kafka connector.
I think `INCREMENTAL` contains `LATEST` `EARLIEST` `FROM-TIMESTAMP`....